### DISCUSSION TITLE
Is this real 0 day vulnerability/ security threat ? Please confirm

### DISCUSSION THREAD

**Discussion by Coder63** [argoproj/argo-cd#8370] [Q&A] [answered]
Hi all, 
Could someone confirm the threat here ?

https://apiiro.com/blog/malicious-kubernetes-helm-charts-can-be-used-to-steal-sensitive-information-from-argo-cd-deployments/ 

https://venturebeat.com/2022/02/03/major-vulnerability-found-in-open-source-dev-tool-for-kubernetes/

**Comment by slotloader** [comment, marked as answer]
Yes, as per a link on the venturebeat article: https://github.com/argoproj/argo-cd/security/advisories/GHSA-63qx-x74g-jcr7
